libgalois/test/graph-compile-lscsr.cpp

```diff
@@ -116,5 +116,16 @@ int main() {
   // ...
   GALOIS_ASSERT(g[8] == 7);
 
+  {
+    uint64_t num_vertices = (1 << 22) + 67;
+    galois::graphs::LS_LC_CSR_Graph<void, void> big(num_vertices);
+    for (uint64_t i = 0; i < num_vertices; ++i) {
+      big.addEdgesTopologyOnly(i, {(1 + 1) % num_vertices});
```

Comment:
is this supposed to be `(1 + 1)`?

Reply:
lol yes but it does not matter since the prefix sum just computes based on the number of edges.